src/Routers/UsersRouter.js

```diff
@@ -57,6 +57,17 @@ export class UsersRouter extends ClassesRouter {
           const user = response.results[0].user;
           // Send token back on the login, because SDKs expect that.
           user.sessionToken = sessionToken;
+
+          // Remove hidden properties.
+          for (var key in user) {
```

Comment:
There does seem to be handling somewhere that removes hidden fields (see updated test where users/<user object id> is retrieved without hidden field).

The general case where all hidden fields are removed from all unauthenticated queries is likely better to be talked in its own issue, rather than tacked on to this PR.

Does this current version require any further changes?